test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -2963,7 +2967,7 @@ BOOST_AUTO_TEST_CASE(cyclic_binary_dependency_via_inheritance)
 BOOST_AUTO_TEST_CASE(multi_variable_declaration_fail)
 {
 	char const* text = R"(
-		contract C { function f() { var (x,y); } }
+		contract C { function f() { var (x,y); x;y;} }
 	)";
 	CHECK_ERROR(text, TypeError, "");
```

Comment:
Please add the correct error message if you change this, so it will not change to a different error without us noticing.